src/mlpack/methods/linear_regression/linear_regression.cpp

```diff
@@ -141,7 +141,7 @@ double LinearRegression::ComputeError(const arma::mat& predictors,
       Log::Fatal << "The test data must have the same number of columns as the "
           "training file." << std::endl;
     }
-    temp = responses - arma::trans(parameters) * predictors;
+    temp = responses - (arma::trans(parameters) * predictors);
```

Comment:
I think the previous style is just fine, no need to explicitly add ``()`` around a multiplication.

Reply:
yes it is fine due to precedence but I still think that those braces adds some level of readability to such mathematical expressions and removes extra overhead of remembering precedence especially for beginners

Reply:
Due to some problem from my side I am closing this PR and created a new PR for all documentation correction here #1792